Institutional Support (#1880)
trixi-framework/Trixi.jl: README.md

[join us on Slack](https://join.slack.com/t/trixi-framework/shared_invite/zt-sgkc6ppw-6OXJqZAD5SPjBYqLd8MU~g)
or [create an issue](https://github.com/trixi-framework/Trixi.jl/issues/new).

## Institutional Support

> Trixi.jl relies on institutions that provide infrastructure for coding and testing, as well as encourage students, post-docs and scientists to contribute code. Institutes and groups that supported us in the past and present are listed below.

Reviewer: We try to keep the line length limited to make it easier to review changes in git. You suggested using a "meaningful order". I'm not sure which order would be best. We could start with a chronological order since that requires the least amount of changes when updating the list in the future. However, I am of course open to other suggestions.

```suggestion
Trixi.jl relies on institutions that provide infrastructure for coding and testing,
as well as encourage students, post-docs, and scientists to contribute code.
Some institutes and groups that supported us in the past and present are listed below,
roughly in chronological order.
```

<p align="left">
  <img style="float:left;height:30px;margin-right:10px" 
    src="https://github.com/torrilhon/Trixi.jl/assets/17232977/e9432f4c-995f-434e-b393-d15efe68abf1">
  <a href="https://www.mi.uni-koeln.de/NumSim/">Numerical Simulation Group, University of Cologne</a>

> </p>
> 
> <p align="left">

Reviewer: Here are some additional institutes that have supported us significantly (in chronological order as far as I remember)

```suggestion
</p>

- [Applied Mathematics, Linköping University, Sweden](https://liu.se/en/organisation/liu/mai/tima)
- [Applied Mathematics, University of Münster, Germany](https://www.uni-muenster.de/AMM/en/)
- [Computational and Applied Mathematics, Rice University, USA](https://cmor.rice.edu)
- [Applied Mathematics, University of Hamburg, Germany](https://www.math.uni-hamburg.de/en/forschung/bereiche/am.html)

<p align="left">
```

We should add their logos etc. as well, of course

Reviewer 2: Shouln't Uni Mainz be included in this list?

Reviewer: It is - but I moved to Mainz after people from Aachen started contributing, I think 🤔 That's why I have added Mainz below.

Reviewer 2: Whoops, I missed that, my bad.

  <img style="float:left;height:30px;margin-right:10px" 
    src="https://github.com/torrilhon/Trixi.jl/assets/17232977/c3e1f6c9-4d13-4001-b817-b3ab405f177b">
  <a href="https://www.acom.rwth-aachen.de">Lab for Applied and Computational Mathematics, RWTH Aachen</a>

> </p>
>   
> ...and many more.

Reviewer: continuing...

```suggestion
</p>

- [Numerical Mathematics, Institute of Mathematics, Johannes Gutenberg University Mainz, Germany](https://www.nummath.math.uni-mainz.de)
```

[join us on Slack](https://join.slack.com/t/trixi-framework/shared_invite/zt-sgkc6ppw-6OXJqZAD5SPjBYqLd8MU~g)
or [create an issue](https://github.com/trixi-framework/Trixi.jl/issues/new).

> ## Institutional Support
> Trixi.jl relies on institutions that provide infrastructure for coding and testing,
> as well as encourage students, post-docs, and scientists to contribute code.
> Some institutes and groups that supported us in the past and present are listed below,
> roughly in chronological order.
> 
> <p align="left">
>   <img style="float:left;height:30px;margin-right:10px" 
>     src="https://github.com/torrilhon/Trixi.jl/assets/17232977/e9432f4c-995f-434e-b393-d15efe68abf1">
>   <a href="https://www.mi.uni-koeln.de/NumSim/">Numerical Simulation Group, University of Cologne</a>
> </p>
> <p align="left">
>   <img style="float:left;height:30px;margin-right:10px" 
>     src="https://github.com/torrilhon/Trixi.jl/assets/17232977/387319ba-9e08-4ad8-9154-628416b247b6">
>   <a href="https://liu.se/en/organisation/liu/mai/tima">Applied Mathematics, Linköping University, Sweden</a>
> </p>
> <p align="left">
>   <img style="float:left;height:30px;margin-right:10px" 
>     src="https://github.com/torrilhon/Trixi.jl/assets/17232977/e2b9e54b-fe00-40f8-b6a1-1152ed677d0a">
>   <a href="https://www.uni-muenster.de/AMM/en/">Applied Mathematics, University of Münster, Germany</a>
> </p>
> <p align="left">
>   <img style="float:left;height:30px;margin-right:10px" 
>     src="https://github.com/torrilhon/Trixi.jl/assets/17232977/17f64669-c581-4788-b20c-7a98055b743b">
>   <a href="https://cmor.rice.edu">Computational and Applied Mathematics, Rice University, USA</a>
> </p>
> <p align="left">
>   <img style="float:left;height:30px;margin-right:10px" 
>     src="https://github.com/torrilhon/Trixi.jl/assets/17232977/34f243bc-cd6c-4935-8b4c-ec9898df59c3">
>   <a href="https://www.math.uni-hamburg.de/en/forschung/bereiche/am.html">Applied Mathematics, University of Hamburg, Germany</a>
> </p>
> <p align="left">
>   <img style="float:left;height:30px;margin-right:10px" 
>     src="https://github.com/torrilhon/Trixi.jl/assets/17232977/c3e1f6c9-4d13-4001-b817-b3ab405f177b">
>   <a href="https://www.acom.rwth-aachen.de">Applied and Computational Mathematics, RWTH Aachen, Germany</a>
> </p>
> <p align="left">
>   <img style="float:left;height:30px;margin-right:10px" 
>     src="https://github.com/torrilhon/Trixi.jl/assets/17232977/7d4c8e93-4efa-40f9-bbe9-f9a5ba52690a">
>   <a href="https://www.nummath.math.uni-mainz.de">Numerical Mathematics, Institute of Mathematics, Johannes Gutenberg University Mainz, Germany</a>
> </p>

Reviewer 3: As per #1989:

```suggestion

## Participating research groups
Participating research groups in alphabetical order:

[Applied and Computational Mathematics, RWTH Aachen University :de:](https://www.acom.rwth-aachen.de)

[Applied Mathematics, Department of Mathematics, University of Hamburg :de:](https://www.math.uni-hamburg.de/en/forschung/bereiche/am.html)

[Applied Mathematics, Linköping University :sweden:](https://liu.se/en/employee/andwi94)

[Computational and Applied Mathematics, Rice University :us:](https://jlchan.github.io/)

[High-Performance Computing, Institute of Software Technology, German Aerospace Center (DLR) :de:](https://www.dlr.de/en/sc/about-us/departments/high-performance-computing)

[High-Performance Scientific Computing, University of Augsburg :de:](https://www.uni-augsburg.de/fakultaet/mntf/math/prof/hpsc)

[Numerical Mathematics, Institute of Mathematics, Johannes Gutenberg University Mainz :de:](https://ranocha.de)

[Numerical Simulation, Department of Mathematics and Computer Science, University of Cologne :de:](https://www.mi.uni-koeln.de/NumSim/)

```
